visualizer/README.md

```diff
@@ -1,15 +1,22 @@
 # MIPT-V Module topology visualizer tool
+Application fetches the topology data generated by the special MIPT-V command line option and visualizes it in an interactive graph
 ## Requirements
 * npm >= 6
 * node >= 8

```

Comment:
The description is too coarse. Again, there is a lack of motivation. Please check how requirements are described in "parent" [readme.md](https://github.com/MIPT-ILab/mipt-mips/blob/master/readme.md)